src/client/services/apiCaller/index.js

```diff
@@ -1,16 +1,43 @@
 const io = require('socket.io-client');
+const qs = require('querystring');
 let socket = io.connect('/');
 
-const CDC = {
+const apiCaller = {
   getGraphAndLayout(uri, version) {
-    return fetch(`/api/get-graph-and-layout?uri=${encodeURIComponent(uri)}&version=${version}`, {
+    return fetch(`/api/get-graph-and-layout?${qs.stringify({uri, version})}`, {
```

Comment:
does qs.stringify encode the uri?

Reply:
It creates a query string from the object and encodes all values